app/controllers/admin/custom_contents_controller.rb

```diff
@@ -26,5 +27,9 @@ def save_current
           setting.save
         end
       end
+
+      def clear_cache
```

Comment:
__[reek]__
* [UtilityFunction] doesn't depend on instance state (maybe move it to another class?) [:ref](https://github.com/troessner/reek/blob/master/docs/Utility-Function.md)

<!-- Comment by [SideCI](https://sideci.com) -->